geode-core/src/main/java/org/apache/geode/internal/cache/wan/serial/SerialGatewaySenderEventProcessor.java

```diff
@@ -111,10 +111,9 @@ public SerialGatewaySenderEventProcessor(AbstractGatewaySender sender, String id
       ThreadsMonitoring tMonitoring, boolean cleanQueues) {
     super("Event Processor for GatewaySender_" + id, sender, tMonitoring);
 
+    initializeMessageQueue(id, cleanQueues);
```

Comment:
This change is because the CacheListener to handle the unprocessedEventsMap is set here, hence I wanted the listener to be up and running before the map is initialized so the no events are missed because they were put in the map before the listener was running.